gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -420,9 +420,7 @@
             <property name="tagOrder" value="@param, @return, @throws, @deprecated"/>
             <property name="target" value="CLASS_DEF, INTERFACE_DEF, ENUM_DEF, METHOD_DEF, CTOR_DEF, VARIABLE_DEF"/>
         </module>
-        <module name="CyclomaticComplexity"> <!-- Java Coding Guidelines: Reduce Cyclomatic Complexity -->
-            <property name="switchBlockAsSingleDecisionPoint">true</property>
-        </module>
+        <module name="CyclomaticComplexity"/> <!-- Java Coding Guidelines: Reduce Cyclomatic Complexity -->
```

Comment:
Revert seems fine, but should we just fix this?

```suggestion
        <module name="CyclomaticComplexity"> <!-- Java Coding Guidelines: Reduce Cyclomatic Complexity -->
            <property name="switchBlockAsSingleDecisionPoint" value="true"/>
        </module>
```

Reply:
also surprised a test didn't catch this, and excavator bump https://github.com/palantir/gradle-baseline/pull/2385 had to catch the [failure](https://app.circleci.com/pipelines/github/palantir/gradle-baseline/4450/workflows/25cc584a-2ef8-4a5a-9e5c-3888a1ef9531/jobs/35168)

```
Caused by: org.xml.sax.SAXParseException; systemId: file:/home/circleci/project/.baseline/checkstyle/checkstyle.xml; lineNumber: 406; columnNumber: 63; Attribute "value" is required and must be specified for element type "property".
```